services/api-server/src/simcore_service_api_server/core/settings.py

```diff
@@ -4,31 +4,23 @@
 from typing import Any
 
 from models_library.basic_types import BootModeEnum, LogLevel
-from pydantic import Field, NonNegativeInt, PositiveInt, SecretStr, parse_obj_as
+from pydantic import Field, NonNegativeInt, PositiveInt, SecretStr
 from pydantic.class_validators import validator
 from settings_library.base import BaseCustomSettings
-from settings_library.basic_types import PortInt, VersionTag
 from settings_library.catalog import CatalogSettings
+from settings_library.director_v2 import DirectorV2Settings
 from settings_library.postgres import PostgresSettings
 from settings_library.rabbit import RabbitSettings
 from settings_library.storage import StorageSettings
 from settings_library.utils_logging import MixinLoggingSettings
-from settings_library.utils_service import (
-    DEFAULT_AIOHTTP_PORT,
-    DEFAULT_FASTAPI_PORT,
-    MixinServiceSettings,
-    URLPart,
-)
 from settings_library.utils_session import (
     DEFAULT_SESSION_COOKIE_NAME,
     MixinSessionSettings,
 )
+from settings_library.webserver import WebServerSettings
 
 
-class WebServerSettings(BaseCustomSettings, MixinServiceSettings, MixinSessionSettings):
-    WEBSERVER_HOST: str = "webserver"
-    WEBSERVER_PORT: PortInt = DEFAULT_AIOHTTP_PORT
-    WEBSERVER_VTAG: VersionTag = Field(default="v0")
+class WebServerExtendedSettings(WebServerSettings, MixinSessionSettings):
```

Comment:
I would prefer to rename  the `WebserverSettings`  to `WebserverBaseSettings` and then call this class `WebserverSettings`. I think we use that convention in other places.